RecoPPS/RPixEfficiencyTools/python/ReferenceAnalysisDQMHarvester_cfg.py

```diff
@@ -82,11 +82,11 @@
 #CONFIGURE DQM Saver
 process.dqmEnv.subSystemFolder = "CalibPPS"
 process.dqmSaver.convention = 'Offline'
-process.dqmSaver.workflow = "/CalibPPS/AlignmentGlobal/CMSSW_11_3_0_pre4"
+process.dqmSaver.workflow = "/CalibPPS/AlignmentGlobal/CMSSW_11_3_0_pre4" #TODO

```

Comment:
Especially this 'CalibPPS/AlignmentGlobal' is inherited from somewhere else. Definitely mark this as something to be replaced.